conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosDatabaseOperations.scala

```diff
@@ -490,7 +493,7 @@ object TezosDatabaseOperations extends DefaultDatabaseOperations("tezos") with L
     * Writes baking rights to the database
     * @param bakingRights mapping of hash to endorsingRights list
     */
-  def insertBakingRights(bakingRights: List[BakingRights]): DBIO[Option[Int]] = {
+  def writeBakingRights(bakingRights: List[BakingRights]): DBIO[Option[Int]] = {
```

Comment:
I think we should consider consistent naming - in one of my previous PRs you asked me to change names to begin with `write` when there were upsert operations, here were doing inserts

Reply:
I'm not sure I follow.
I was trying to be consistent with what I thought was a pattern:
all insert operations were initially called `writeX`.

When some upsert operation was beginning to be used I think I decided to highlight them as such to make it clear that existing records would be overwritten.
I didn't chose upsert to reflect the db naming, it was just the idea of "update when present, or insert a new one", as a single word.
I checked the db operations and no insert was named as such... I opted to align these methods with the rest.

If I'm missing something please point it out, and overall we can decide to define together an explicit naming policy and put it somewhere written as a reference

Reply:
`writeAccounts`, `writeBakers`, `writeBakersAndCopyContracts` use `insertOrUpdateAll` - I think you requested here to change update to write, and that's what got me confused:
https://github.com/Cryptonomic/Conseil/pull/797#discussion_r429145776

Reply:
I must be more confused than you are...
I'm fine as long as we got a consistent naming scheme. My memory is probably failing me in this.

Reply:
I agree, we should get consistent naming there 👍

Reply:
I'm reading the comment you mentioned once more...
and it seems consistent with my request to keep naming all methods that writes data as `writeX`, be it insert or upsert alike.
I see that of the existing `upsertX` methods, I did slip once and used `upsertTezosNames`... I should probably change that too.

As I said, I'm ok to differentiate, as long as
 1. we try to keep it consistent
 2. the differentiation is meaningful at the domain level, not at the database level (that would be an implementation leak)

What do you propose?

Reply:
For now we can leave it as it is, but we can mark it as an improvement for the future.

Reply:
I'm opening a separate issue to address the point, which is more involved and probably is not so relevant to this PR anyway